terra-jupyter-aou/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.1.6 - 2021-06-07T19:39:56.024375Z
```

Comment:
@calbach @jmthibault79 FYI -- this was auto-updated, let me know if you have concerns